likeminds-chat-reactnative-integration/ChatSX/components/InputBox/index.tsx

```diff
@@ -1318,10 +1347,10 @@ const MessageInputBox = ({
                 msg: BLOCKED_DM,
               },
             });
-            dispatch({
-              type: EMPTY_BLOCK_DELETION,
-              body: {},
-            });
+            // dispatch({
```

Comment:
remove unneccessary comments or add comment explain why this line of code is commented.